### PR TITLE
AGDIGGER-86 Document Jenkinsfile usage

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by aidenkeating** [comment]
Ping @wei-lee @odra for review.

File: docs/digger/developer/jenkinsfile.asciidoc [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+----
+
+The `signAndroidApks` function takes the following parameters.
+
+.signAndroidApks options
+|===
+| Parameter | Description
+
+| keyStoreId
+| Keystore ID. This will be provided through the `BUILD_CREDENTIAL_ID` parameter.
```
Comment (wei-lee) [inline docs/digger/developer/jenkinsfile.asciidoc]: Add a note here to explain that this parameter is only provided if the app is built from RHMAP

File: docs/digger/developer/jenkinsfile.asciidoc [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+| keyAlias
+| Alias of the private key/certificate chain. This will be provided through the `BUILD_CREDENTIAL_ALIAS` parameter.
```
Comment (wei-lee) [inline docs/digger/developer/jenkinsfile.asciidoc]: same as above

File: docs/digger/developer/jenkinsfile.asciidoc [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+.codeSign options
+|===
+| Parameter | Description
+
+| profileId
+| ID of the developer profile.
+
+| clean
+| Clean the workspace before signing the app.
```
Comment (odra) [inline docs/digger/developer/jenkinsfile.asciidoc]: I thinks its better to say that it removes the previous signature (if any) before signing the artifact.

File: getstarted/guides/index.markdown [hunk 4]
```diff
@@ -108,4 +108,6 @@ For more details on the supported mobile platforms consult the guides below:
 <h4 id="diggerGuides">Guides</h4>
 * [Installation Guide](/docs/digger/installation)
 * [Admin Guide](/docs/digger/admin)
-* [Developer Guide](/docs/digger/developer)
\ No newline at end of file
+* [Developer Guide](/docs/digger/developer)
+* [Admin Guide](/)
+* [Developer Guide](/docs/digger/developer)
```
Comment (wei-lee) [inline getstarted/guides/index.markdown]: I think there are some duplicated entries here.

File: docs/digger/developer/jenkinsfile.asciidoc [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+| flags
+| Flags to add to the command that is executed.
+|===
+
+.codeSign options
+|===
+| Parameter | Description
+
+| profileId
+| ID of the developer profile.
```
Comment (wei-lee) [inline docs/digger/developer/jenkinsfile.asciidoc]: this value will be passed in as `params.BUILD_CREDENTIAL_ID`

Comment (odra) [inline docs/digger/developer/jenkinsfile.asciidoc]: I think we could mention that our digger jenkins client can send this value as a build parameter

File: getstarted/guides/index.markdown (hunk 4 above)
Comment (aidenkeating) [inline getstarted/guides/index.markdown]: The one time I use a UI-based rebasing tool :joy:

**Comment by aidenkeating** [comment]
@wei-lee @odra I think I got them all

**Comment by wei-lee** [comment]
LGTM

**Pull request merged**